src/core/model.js

```diff
@@ -376,9 +384,21 @@ export default class Model extends Object3D {
         program: this.program,
         uniforms: Object.assign({}, this.uniforms, uniforms)
       });
+
       log.table(priority, table);
       log.log(priority, `${unusedCount || 'No'} unused uniforms `, unusedTable);
     }
+
+    if (window.__SEER_INITIALIZED__) {
+      const uniformObject = getUniformsObject(
+        this.program,
+        Object.assign({}, this.uniforms, uniforms)
+      );
+
+      if (uniformObject) {
+        seer.indexedListItem('luma.gl', this.id, uniformObject);
```

Comment:
Something about this method name - it is not very intuitive. Consider something like `addInfoToItem` ?